tests-jax/test_1399-from-jax.py

```diff
@@ -0,0 +1,50 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+import pytest  # noqa: F401
+import numpy as np  # noqa: F401
+import cupy as cp  # noqa: F401
+import awkward as ak  # noqa: F401
+import jax
+
+import awkward._v2._connect.jax
+
+awkward._v2._connect.jax.register_jax()
```

Comment:
See my comment on the definition of this function: we'd much rather do without it!